src/pages/home/Home.js

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import {Link} from "react-router-dom";
+import './Home.css'
+
+function Home(props) {
+    const [reddit,setReddit] = useState([]);
+    const controller =  new AbortController();
+    async function fetchHomePage () {
+        try{
+            const response = await axios.get("https://www.reddit.com/hot.json?limit=15")
+            console.log(response.data.data.children)
+            setReddit(response.data.data.children)
+        }
+        catch(e) {
+            console.error(e)
+        }
+    }
```

Comment:
Formeel gezien zou je deze functienaam in de dependency array van de useEffect op moeten nemen. Echter gaat de useEffect dan bij elke rerender af, omdat de functie búiten de useEffect staat en daardoor elke rerender vernieuwd. Je kunt dan twee dingen doen:

1. Je zet deze functie bínnen de useEffect functie
2. (Niet behandeld tijdens de les geloof ik): Je maakt gebruik van de [useCallback hook](https://reactjs.org/docs/hooks-reference.html#usecallback).